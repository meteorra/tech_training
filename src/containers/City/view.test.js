import React from 'react';
import renderer from 'react-test-renderer';
import CityView from './view';

function setup() {
    const props = {
        fetchForecast: jest.fn(),
        code: 'minsk',
        forecast: {},
    };

    const View = CityView(() => null);
    const render = renderer.create(<View {...props}/>);

    return {
        render,
    };
}

describe('CityView', () => {

    it('renders h1 title with code property "Here is minsk weather forecast"', () => {

        const { render, } = setup();
        expect(render.toJSON()).toMatchSnapshot();
    });
});
