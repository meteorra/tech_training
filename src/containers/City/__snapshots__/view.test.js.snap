// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CityView renders h1 title with code property equals minsk "Here is minsk weather forecast" 1`] = `
<div
  className="row"
>
  <div
    className="col-md-8 col-md-offset-2"
  >
    <div
      className="panel panel-default"
    >
      <div
        className="panel-body"
      >
        <h1>
          Here is 
          minsk
           weather forecast
        </h1>
        <hr />
      </div>
    </div>
  </div>
</div>
`;
